barman/cli.py

```diff
@@ -790,7 +808,8 @@ def check(args):
     output.close_and_exit()
 
 
-def diagnose():
+@command()
+def diagnose(args):
```

Comment:
Also, maybe rebase on master to resolve PR conflict (`tests/requirements_minimal.txt`).
This will allow CI to run tests.

Reply:
> Hi, does we need args as it is not used(or maybe in decorator)?
> It also this breaks tests/test_diagnose.py where diagnose() is used.
> maybe set args=[]

It would need to be `args=None` actually. [https://docs.python-guide.org/writing/gotchas/](https://docs.python-guide.org/writing/gotchas/) 

I don't think this particular case would ever cause an actual problem bc diagnose doesn't have any arguments, but may as well set it to None.